Appraisals

```diff
@@ -1,11 +1,15 @@
-appraise "rails3.2" do
-  gem "rails", "~> 3.2.0"
-end
-
```

Comment:
Related. This needs to be put back.

Reply:
Ok, let me see if I can get it to work. Just FYI, Rails 3.2 isn't really supported by a core team anymore: http://guides.rubyonrails.org/maintenance_policy.html#severe-security-issues